<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoNewDialogClass</class>
 <widget class="QDialog" name="DoNewDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>donewdialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QLabel" name="widthlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体_GB2312</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>改变画布宽 ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="highlabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体_GB2312</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>改变画布高</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="widthspinBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>400</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>520</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="heightspinBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>300</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>390</number>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>76</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体_GB2312</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>画布颜色 ：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <zorder>toolButton</zorder>
  <zorder>buttonBox</zorder>
  <zorder>widthlabel</zorder>
  <zorder>highlabel</zorder>
  <zorder>widthspinBox</zorder>
  <zorder>heightspinBox</zorder>
  <zorder>label</zorder>
  <zorder>textBrowser</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>textBrowser</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
